ci/script.sh

```diff
@@ -23,9 +18,42 @@ inspect() {
     set +e
     $PREFIX$OBJDUMP -Cd target/**/release/*.rlib | gist_it
     set -e
+}
+
+run_tests() {
+    if [[ $QEMU_LD_PREFIX ]]; then
+        export RUST_TEST_THREADS=1
+    fi
+
+    if [[ $RUN_TESTS == y ]]; then
+        cargo test --target $TARGET
+        cargo test --target $TARGET --release
+    fi
+}
+
+c_test() {
+    $CARGO clean
+
+    # verify we haven't drop any intrinsic/symbol
+    $CARGO build --features c --target $TARGET --bin intrinsics
```

Comment:
Only -nostartfiles is needed on Linux. I think that would work on Linux, but idk about OSX and Windows. I'm using the libc crate because, on each platform, it links to different libraries needed to produce an executable. Without it the executable may not link at all.